.github/workflows/automated-testing.yml

```diff
@@ -94,7 +94,7 @@ jobs:
           BRANCH_NAME="${PATHS[1]}_${PATHS[2]}"
           echo "BRANCH=$(echo ${BRANCH_NAME})" >> $GITHUB_ENV
       - name: Coverage Badge
-        uses: schneegans/dynamic-badges-action@v1.0.0
+        uses: schneegans/dynamic-badges-action@v1.7.0
```

Comment:
This change now makes the jobs fail.
The problem seems to be that the authentication fails. This isn't caused by using the newer version, the older version v1.0.0 didn't raise an error if writing to the gist failed. As visible in for example this [3-week-old job log](https://github.com/OpenEnergyPlatform/ontology/actions/runs/10739131014/job/29784581378) the same step in the job logs an error:
```
Run schneegans/dynamic-badges-action@v1.0.0
result:{"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
```
The job only doesn't fail, due to no actual error being raised.

Reply:
Thanks. Is this a problem that we can solve or is it due to an external repo that we use?

Reply:
It uses GitHub Gist to store information about the coverage and currently uses this gist of @areleu: https://gist.github.com/areleu/6d00affa9fbc89c79684d62091d96551
According to this action [`schneegans/dynamic-badges-action`](https://github.com/Schneegans/dynamic-badges-action), a Personal Access Token with the scope `gist` has to be created and added as an organization secret like you did in https://github.com/OpenEnergyPlatform/ontology/pull/1936 with `secrets.GITHUB_TOKEN`.
I assume that the token behind `secrets.GIST_SECRET` ran out. So either a new token is created for the existing gist by @areleu or a new gist is created.

Reply:
I don't quite remember merging this to the main Repo, I had it in my personal branch to test it out and never really implemented it in here because I did not have the rights to add secrets, so I would say that the solution is to create a new gist/token that is specific to this repo. Not sure if gists can be created in a repository or they have to always be from an account. I could just send you a new key for that particular gist if we want to keep it simple

Reply:
I don't think gists can be created for a repository or an organization.

> I could just send you a new key for that particular gist if we want to keep it simple

Yeah, that seems like the easiest solution. I can't access the organizations secrets either, so you would have to send it to @stap-m. :)

Reply:
I created a new secret, I can update it with @stap-m later.

Reply:
Secret should be updated now